paperqa/llms.py

```diff
@@ -50,8 +60,24 @@ class EmbeddingModes(StrEnum):
     QUERY = "query"
 
 
+CHARACTERS_PER_TOKEN: float = 4.0
```

Comment:
Can you name this `CHARACTERS_PER_TOKEN_ASSUMPTION` and add a comment saying why we chose `4`